packages/data-stores/src/user/actions.ts

```diff
@@ -9,9 +9,7 @@ import {
 	NewUserSuccessResponse,
 } from './types';
 
-export const fetchCurrentUser = () => ( {
-	type: ActionType.FETCH_CURRENT_USER as const,
-} );
```

Comment:
Select `getCurrentUser`, I don't believe we need this.

Reply:
Good point, I don't think we need it yet, either — I added it in in case we wanted a fetching state for checking if there's a current user, but I can't quite think of a real use case. E.g. when Gutenboarding loads, we can assume it's in a loading state by default (where `currentUser` is undefined) and after the resolver fires `currentUser` will either be a user object or `null` if not logged in, so the fetching state could be inferred.

Reply:
I left a relevant note about that here: https://github.com/Automattic/wp-calypso/pull/38823#discussion_r368490804 (sorry for disconnect). A selector+resolver should have without any additional code.